pkg/controllers/terminating/v1alpha1/terminate.go

```diff
@@ -92,7 +99,27 @@ func (t *Terminator) terminate(ctx context.Context, node *v1.Node) error {
 	return nil
 }
 
-// getPods returns a list of pods scheduled to a node
+// evictPods returns true if there are no evictable pods
+func (t *Terminator) evictPods(ctx context.Context, pods []*v1.Pod) bool {
```

Comment:
I think we need to make this function return (bool,error) and differentiate between a 429 (backoff) and other errors like 403.

Reply:
I don't know if that's quite necessary as both would require an exponential backoff and retry. Anytime we would be returning false here, we would need to retry anyways. I can see in the future as we implement more robust eviction logic that this might be necessary, but as we don't want to block all evictions on one failed eviction, this works right now.